docs/getting_started.md

```diff
@@ -73,6 +73,48 @@ namespace :admin do
 end
 ```
 
+## Keep Dashboards Updated as Model Attributes Change
+
+If you've installed Administrate and generated dashboards and _then_ 
+subsequently added attributes to your models you'll need to manually add 
+these additions (or removals) to your dashboards.
+
+Example:
+
+```ruby
+# app/dashboards/your_model_dashboard.rb
+
+  ATTRIBUTE_TYPES = {
+    # ...
+    the_new_attribute: Field::String,
+    # ...
+  }.freeze
+  
+  SHOW_PAGE_ATTRIBUTES = [
+    # ...
+    :the_new_attribute,
+    # ...
+  ].freeze
+
+  FORM_ATTRIBUTES = [
+    # ...
+    :the_new_attribute,
+    # ...
+  ].freeze
+  
+  COLLECTION_ATTRIBUTES = [
+    # ...
+    :the_new_attribute, # if you want it on the index, also.
+    # ...
+  ].freeze
+```
+
+It is recommended you make this change in the PR that also adds the attribute to 
+your model.
```

Comment:
I wonder if we'd be better off not being prescriptive in the process here? (Even though I do _exactly_ this.) What do you think about changing this to be:

> It's recommended that you make this change at the same time as you add the attribute to the model

Reply:
Yup, good point. will change this.